Server/Utilities/SpringBoardAlerts.m

```diff
@@ -1,7 +1,27 @@
 
+#import <UIKit/UIKit.h>
```

Comment:
Do we need UIKit because of NSBundle and NSDataAsset?

Reply:
UIKit is needed for NSDataAsset